__tests__/app.test.js

```diff
@@ -207,6 +228,19 @@ describe("POST: /api/articles/:article_id/comments", () => {
                 expect(body).toEqual({ msg: "Not found" });
             })
     })
+    test("404: Error - should return Not found when passing a comment with no username", () => {
```

Comment:
I would make this description clearer - a comment with a username that does not exist